resources/dicts/events/death/general.json

```
@@ -0,0 +1,5870 @@
+        "location": ["beach", "mountainous", "forest"],
+        "season": [
+            "any"
+        ],
+        "sub_type": ["murder"],
+        "tags": [],
+        "weight": 20,
+        "event_text": "m_c was pushed off a cliff onto the sharp rocks at the bottom. r_c stands above {PRONOUN/m_c/object}, watching as they fall.",
+        "m_c": {
+            "status": [ "apprentice", "mediator apprentice", "medicine cat apprentice", "warrior", "medicine cat", "mediator", "deputy"],
+            "not_skill": [
+                "PROPHET,3",
+                "OMEN,3",
+                "FIGHTER,3"
+            ],
+            "dies": true
+        },
+        "r_c": {
+            "age": [ "adolescent", "young adult", "adult", "senior adult" ],
+            "skill": [
+                "HUNTER,3",
+                "FIGHTER,2"
+            ]
+        },
+        "history": [{
+            "cats": ["m_c"],
+            "reg_death": "m_c was pushed off a cliff by r_c."
+        }],
+        "relationships": [
+            {
+                "cats_from": [
+                    "r_c"
+                ],
+                "cats_to": [
+                    "m_c"
+                ],
+                "values": [
+                    "dislike"
+                ],
+                "amount": 15
+            },
+            {
+                "cats_from": [
+                    "r_c"
+                ],
+                "cats_to": [
+                    "m_c"
+                ],
+                "values": [
+                    "platonic",
+                    "trust",
+                    "comfort"
+                ],
+                "amount": -15
+            }
+        ]
+    },
+    {
+        "event_id": "gen_death_war_any3",
+        "location": ["mountainous", "forest", "beach"],
+        "season": [
+            "any"
+        ],
+        "sub_type": [ "war" ],
+        "tags": [ "some_lives"],
+        "weight": 20,
+        "event_text": "m_c was killed by a falling boulder pushed from a cliff above {PRONOUN/m_c/object} by o_c_n warriors.",
+        "m_c": {
+            "status": [ "apprentice", "warrior", "deputy", "leader" ],
+            "dies": true
+        },
+        "history": [{
+            "cats": ["m_c"],
+            "reg_death": "m_c was crushed by a falling boulder.",
+            "lead_death": "{VERB/m_c/were/was} crushed by a falling boulder."
+        }],
+        "other_clan": {
+            "current_rep": [
+                "hostile"
+            ],
+            "changed": -3
+        }
+    },
+    {
+        "event_id": "gen_death_hawk_any1",
+        "location": ["any"],
+        "season": [
+            "any"
+        ],
+        "tags": ["no_body"],
+        "weight": 20,
+        "event_text": "r_c, guarding camp, yowls a warning as they spot a circling shadow, right below m_c. r_c starts sprinting to m_c's position, but couldn't get there in time, not before m_c was carried high into the sky.",
```

```suggestion
        "event_text": "r_c, guarding camp, yowls a warning as they spot a circling shadow, right above m_c. r_c starts sprinting to m_c's position but couldn't get there in time, not before m_c was carried high into the sky.",
```

At least, I think it should be above?